src/apps/components/itemPropertiesConfig/ManeuverConfiguration.svelte

```diff
@@ -33,130 +33,120 @@
-                        />
-
-                        <label class="u-pointer" for={`${appId}-isStance`}>
-                            Manuever is a stance
-                        </label>
-                    </div>
-                </FormSection>
-
-                <FormSection>
-                    <div
-                        class="u-align-center u-flex u-flex-wrap u-gap-md u-text-sm u-w-full"
-                    >
-                        <label
-                            class="u-text-bold u-w-full"
-                            for={`${appId}-exertion-cost`}
-                        >
-                            {localize("A5E.ItemExertionCost")}
-                        </label>
-
-                        <div class="u-w-20">
-                            <input
-                                type="number"
-                                data-dtype="Number"
-                                name="system.exertionCost"
-                                value={$item.system.exertionCost}
-                                id={`${appId}-exertion-cost`}
-                                on:change={({ target }) =>
-                                    updateDocumentDataFromField(
-                                        $item,
-                                        target.name,
-                                        Number(target.value)
-                                    )}
-                            />
-                        </div>
-                    </div>
-                </FormSection>
-            {/if}
-        </div>
-    {:else}
-        <dl class="a5e-box u-flex u-flex-col u-gap-sm u-m-0 u-p-md u-text-sm">
-            <div class="u-flex u-gap-md">
-                <dt class="u-text-bold">
-                    {localize("A5E.ManeuverDegreePrompt")}:
-                </dt>
-
-                <dd class="u-m-0 u-p-0">
-                    {localize(A5E.maneuverDegrees[$item.system.degree])}
-
-                    {#if $item.system.degree > 0 && $item.system.isStance}
-                        (Stance)
-                    {/if}
-                </dd>
+      </FormSection>
+
+      {#if $item.system.degree > 0}
+        <FormSection heading="A5E.ManeuverTraditionPrompt">
+          <RadioGroup
+            options={Object.entries(A5E.maneuverTraditions)}
+            selected={$item.system.tradition}
+            name="system.tradition"
+            document={item}
+          />
+        </FormSection>
+
+        <FormSection>
+          <div class="u-align-center u-flex u-gap-md">
+            <input
+              class="u-pointer"
+              type="checkbox"
+              name="system.isStance"
+              id={`${appId}-isStance`}
+              checked={$item.system.isStance}
+              on:change={({ target }) =>
+                updateDocumentDataFromField($item, target.name, target.checked)}
+            />
+
+            <label class="u-pointer" for={`${appId}-isStance`}>
+              Manuever is a stance
```

Comment:
This needs localization.

Reply:
It does, but that wasn't added by Josh. I'll be making this change while doing the item summaries and so on.